src/controller/userController.ts

```diff
@@ -3,10 +3,22 @@ import * as UserService from '../services/userService'
 
 export const getAllUsers = async (_req: Request, res: Response): Promise<void> => {
   const allUser = await UserService.getAllUsers()
+  if ('code' in allUser) {
+    if (allUser.code === 404) {
+      res.status(404)
+    }
+  }
   res.send(allUser)
 }
 
 export const getOneUserById = async (req: Request, res: Response): Promise<void> => {
-  // TODO: need implementation
-  res.send(req.params.id)
+  const userById = await UserService.getUserById(req.params.id)
+  if ('code' in userById) {
+    if (userById.code === 404) {
+      res.status(404)
+    } else if (userById.code === 500) {
```

Comment:
no need 500 error code...
We might only use 401 (Unauthorized) and 404 (not found)

Reply:
500 codes changed to 401 (logic will need changing in the future probably)